lwt/cohttp_lwt_unix.ml

```diff
@@ -75,7 +75,9 @@ module Server = struct
       Lwt_stream.on_terminate stream (fun () ->
         ignore_result (Lwt_io.close ic));
       let body = Cohttp_lwt_body.of_stream stream in
-      let res = Cohttp.Response.make ~status:`OK ~encoding ?headers () in
+      let mime_type = Magic_mime.lookup fname in
+      let headers = Cohttp.Header.add_opt headers "content-type" mime_type in
```

Comment:
fixed in bc7438d093cbf90b2911dcbec37017794fb22cd0 and new header function in d00503b48e5162e24b12135852db37409ae866e8